### PR TITLE
Provides a config to expose Alluxio master URL to support K8s Env

### LINKED ISSUE #7985: [FEA] Expose Alluxio master URL to support K8s Env
**Is your feature request related to a problem? Please describe.**
## Problem:
  Currently Plugin reads Alluxio master URL from local file. It's controled by `spark.rapids.alluxio.home` config now. Currently we assume the Spark driver and Alluxio master are co-located.
  But on the K8s Env,  Alluxio cluster maybe deployed separately from Spark cluster, then we can't read Alluxio master URL from local file.

**Describe the solution you'd like**
Add a new config to specify the Alluxio master

### PR DESCRIPTION
**Pull request by res-life**
Closes #7985

## Problem:
  Currently Plugin reads Alluxio master URL from local file. It's controled by `spark.rapids.alluxio.home` config now.
  But for the K8s Env and Alluxio cluster is deployed separately, we can't read Alluxio master URL from local file.

## Solution:
  Provides a config `spark.rapids.alluxio.master` to specify the remote Alluxio master URL.
  If has this config, first use this config, otherwise follows the original logic.

Signed-off-by: Chong Gao <res_life@163.com>

### CODE REVIEW

**Comment by res-life** [comment]
build

**Comment by res-life** [comment]
build

File: sql-plugin/src/main/scala/com/nvidia/spark/rapids/AlluxioConfigReader.scala [hunk 1]
```diff
@@ -27,7 +27,14 @@ import scala.io.BufferedSource
 class AlluxioConfigReader {
 
   def readAlluxioMasterAndPort(conf: RapidsConf): (String, String) = {
-    readMasterAndPort(conf.getAlluxioHome)
+    if(conf.getAlluxioMaster.isEmpty) {
```
Comment (tgravescs) [inline sql-plugin/src/main/scala/com/nvidia/spark/rapids/AlluxioConfigReader.scala:30]: nit, add a space after if

File: sql-plugin/src/main/scala/com/nvidia/spark/rapids/AlluxioConfigReader.scala [hunk 2]
```diff
@@ -27,7 +27,14 @@ import scala.io.BufferedSource
...
+      // Read from local Alluxio home
+      readMasterAndPort(conf.getAlluxioHome)
+    } else {
+      // Alluxio master separately deployed and not co-located with Spark Driver.
+      // Like: EKS Env,
+      (conf.getAlluxioMaster, "19998")
```
Comment (tgravescs) [inline sql-plugin/src/main/scala/com/nvidia/spark/rapids/AlluxioConfigReader.scala:36]: seems odd to assume port is 19998, we should either add another config for port as well

**Review by tgravescs — APPROVED** [review]

File: sql-plugin/src/main/scala/com/nvidia/spark/rapids/AlluxioConfigReader.scala (hunk 2 above)
Comment (res-life) [reply sql-plugin/src/main/scala/com/nvidia/spark/rapids/AlluxioConfigReader.scala]: Done

File: sql-plugin/src/main/scala/com/nvidia/spark/rapids/AlluxioConfigReader.scala (hunk 1 above)
Comment (res-life) [reply sql-plugin/src/main/scala/com/nvidia/spark/rapids/AlluxioConfigReader.scala]: Done

**Review by tgravescs — APPROVED** [review]

**Pull request merged**